R/help.R

```diff
@@ -323,11 +336,22 @@ arg_descriptions_from_doc_default <- function(args, doc) {
 
 # Extract arguments descriptions for docs in Sphinx style
 arg_descriptions_from_doc_sphinx <- function(doc) {
+  
   doctree <- sphinx_doctree_from_doc(doc)
   params <- doctree$ids$parameters$children[[2]]$children
-  sapply(params, function(param) {
-    param$children[[3]]$astext()
+  
+  output <- lapply(params, function(param) {
+    children <- param$children
+    children[[length(children)]]$astext()
   })
+  
+  nms <- vapply(params, function(param) {
+    name <- param$children[[1]]$astext()
```

Comment:
It looks like the first entry can capture additional 'stuff' (e.g. the type declaration) so I think we need to strip that off.